### ISSUE TITLE
HeishaMon big one and Eth connetion problem

### ISSUE THREAD

**Issue by Udo3418** [heishamon/HeishaMon#593]
I have installed a HeishaMon big one and can't get it to be addressed via the Eth interface. 
What do I have to set to switch the WiFi function to Eth? 
WiFi works problem-free so far.
V 3.8

**Comment by Binifada1956** [comment]
Do you have the Wifi-Hotspot in the Settings disabled?

**Comment by IgorYbema** [comment]
It just works plug and play. It should get a IP address from your network automatically if you connect the ethernet cable to it. 
Does the module green and yellow leds work?

**Comment by Udo3418** [comment]
No WiFi SSID and No Pw set.
Eth modul Led green and yellow blinking
on the same 2 port switch is the Panasonic Adapter working 
fritz.box can´t find new ip for heisamon
is this automaticly with Dhcp ?

**Comment by IgorYbema** [comment]
Yes should get IP using DHCP. Does not depend on any wifi setting. 
Do you see the wifi network? Is the colored LED on heishamon yellow or blinking blue? Or is it turned off? If it is turned off and you don't see the wifi hotspot, it could be that the heishamon board doesn't have the heishamon firmware yet. Probably failed to install before shipping

**Comment by Udo3418** [comment]
check this tomorrow it´s raining outside

**Comment by Udo3418** [comment]
Power green
Status Led blue on not blinking
Eth Adapter yellow blinking

**Comment by IgorYbema** [comment]
If your status LED is always BLUE this means that is it logging a bunch of data. One BLUE blink is a log line which is printed to the console.  Maybe check the logs for a clue what is happening? Maybe this is causing heishamon not getting an IP address

**Comment by rvdgaag** [comment]
My best guess, issue caused by the limited "switching" capacity of the Fritzbox. Pull a cable to an individual port to the router separately from the Panasonic controller board, ie remove the 2 port switch
